### DISCUSSION TITLE
PAC3 Projectile Suggestion: remove on collide with player

### DISCUSSION THREAD

**Discussion by ghost** [CapsAdmin/pac3#481] [Ideas]
As said in the title above, the feature would work like this, the projectile will be on the ground, staying there, then a player tries to reach the projectile, the projectile disappears and the player gets hurt, or at least get some health.

TL;DR the projectile will get removed once it touches a player and nothing else.

**Comment by lilggamegenius** [comment]
like a mine?

**Comment by ghost** [comment]
Yeah, like a mine.
